tests/ci/cdk/cdk/linux_docker_image_batch_build_stack.py

```diff
@@ -63,12 +63,4 @@ def __init__(self, scope: core.Construct, id: str, **kwargs) -> None:
             role=role,
             timeout=core.Duration.minutes(120),
             build_spec=codebuild.BuildSpec.from_object(build_spec_content))
-
-        # Add 'BuildBatchConfig' property, which is not supported in CDK.
-        # CDK raw overrides: https://docs.aws.amazon.com/cdk/latest/guide/cfn_layer.html#cfn_layer_raw
-        # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codebuild-project.html#aws-resource-codebuild-project-properties
-        cfn_build = project.node.default_child
-        cfn_build.add_override("Properties.BuildBatchConfig", {
-            "ServiceRole": role.role_arn,
-            "TimeoutInMins": 120
```

Comment:
We may want to carry on this because this dimension is not checked in each PR and there will be more docker images build.

Reply:
Yeah I agree, we can update the timeout limit of this.